whiteblock/cmd/transactions.go

```diff
@@ -86,6 +105,33 @@ Optional Parameters:
 	},
 }
 
+var sendToTxCmd = &cobra.Command{
+	// Hidden: true,
+	Use: "to",
+	Short: "Send transaction data to an account",
+	Long: `
+The user must specify the flags that will be used for sending transaction data.
+Send a transaction data to an account.
+
+Required Parameters: 
+	--destination <address> --value <amount> --data <transaction data>
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+		command := ""
+		params := []string{}
+
+		if !(len(toFlag) > 0) || valueFlag == 0 || !(len(dataFlag) > 0)  {
+			util.Print("Required flags were not provided. Please input the required flags.")
+			cmd.Help()
+			return
+		}
+		command = "eth::send_to"
+		params = []string{toFlag, strconv.Itoa(valueFlag), dataFlag}
```

Comment:
No need for conversions, params is actually []interface{}, so use a slice of interface, and don't convert. Also, might want to avoid using the global flag, as it is currently pretty gross. Plus in this case, since it is not being converted to ether and is in wei, you want to use a string. Rationale is that a uint64 can hold a maximum value equal to around 18 eth. And they might want to send more.